wp-content/themes/core/components/blocks/stats/Stats_Block_Controller.php

```diff
@@ -0,0 +1,162 @@
+<?php
+declare( strict_types=1 );
+
+namespace Tribe\Project\Templates\Components\blocks\stats;
+
+use Tribe\Libs\Utils\Markup_Utils;
+use \Tribe\Project\Blocks\Types\Stats\Stats as Stats_Block;
+use Tribe\Project\Templates\Components\Abstract_Controller;
+use \Tribe\Project\Templates\Components\statistic\Statistic_Controller as Statistic;
+use \Tribe\Project\Templates\Components\content_block\Controller as Content_Block;
+
+/**
+ * Class Hero
+ */
+class Stats_Block_Controller extends Abstract_Controller {
+	public const LAYOUT            = 'layout';
+	public const DISPLAY_DIVIDERS  = 'display_dividers';
+	public const CONTENT_ALIGN     = 'content_align';
+	public const TITLE             = 'title';
+	public const DESCRIPTION       = 'description';
+	public const CONTAINER_CLASSES = 'container_classes';
+	public const CONTENT_CLASSES   = 'content_classes';
+	public const CLASSES           = 'classes';
+	public const ATTRS             = 'attrs';
+	public const STATS             = 'stats';
+
+	public string $layout;
+	public string $content_align;
+	public string $display_dividers;
+	public string $title;
+	public string $description;
+	public array $container_classes;
+	public array $content_classes;
+	public array $classes;
+	public array $attrs;
+	public array $stats;
+
+	public function __construct( array $args = [] ) {
+		$args = $this->parse_args( $args );
+
+		$this->layout            = (string) $args[ self::LAYOUT ];
+		$this->content_align     = (string) $args[ self::CONTENT_ALIGN ];
+		$this->display_dividers  = (string) $args[ self::DISPLAY_DIVIDERS ];
+		$this->title             = (string) $args[ self::TITLE ];
+		$this->description       = (string) $args[ self::DESCRIPTION ];
+		$this->container_classes = (array) $args[ self::CONTAINER_CLASSES ];
+		$this->content_classes   = (array) $args[ self::CONTENT_CLASSES ];
+		$this->classes           = (array) $args[ self::CLASSES ];
+		$this->attrs             = (array) $args[ self::ATTRS ];
+		$this->stats             = (array) $args[ self::STATS ];
+	}
+
+	/**
+	 * @return array
+	 */
+	protected function defaults(): array {
+		return [
+			self::LAYOUT            => Stats_Block::LAYOUT_STACKED,
+			self::CONTENT_ALIGN     => Stats_Block::CONTENT_ALIGN_LEFT,
+			self::DISPLAY_DIVIDERS  => Stats_Block::DISPLAY_DIVIDERS,
+			self::TITLE             => '',
+			self::DESCRIPTION       => '',
+			self::CONTAINER_CLASSES => [ 'b-stats__container', 'l-container' ],
+			self::CONTENT_CLASSES   => [ 'b-stats__content' ],
+			self::CLASSES           => [ 'c-block', 'b-stats' ],
+			self::ATTRS             => [],
+			self::STATS             => [],
+		];
+	}
```

Comment:
Dave may have already called this out so forgive me, but I'd set defaults for the 3 class args to be empty arrays and then utilize the `required()` method and implement what you have there as required base classes